src/components/schemas/fields.yaml

```diff
@@ -691,3 +691,11 @@ bank_ifsc:
   description: >
       IFSC code of the bank account of the Employee
   pattern: '^[A-Z]{4}0[A-Z0-9]{6}$'
+
+
+is_implicit_merge_blocked:
+  type: boolean
+  nullable: true
+  description: |
+    This indicates if the expense can be considered for implicit merge or not
```

Comment:
`This indicates if the expense can be merged implicitly or not`
^ My suggestion, if it doesn't break the context then can add this.

Reply:
Done